tony-core/src/main/java/com/linkedin/tony/LocalizableResource.java

```diff
@@ -42,6 +45,14 @@ public boolean isDirectory() {
     return isDirectory;
   }
 
+  public boolean isLocalFile() {
+    return new Path(path).toUri().getScheme() == null;
+  }
+
+  public boolean isArchive() {
+    return resourceType == LocalResourceType.ARCHIVE;
+  }
+
   public Path getSourceFilePath() {
     return sourceFilePath;
   }
```

Comment:
(I wasn't able to comment directly on line 66, so I'm commenting here.) Indentation is off on line 66.